color_refiment_helper.py

```diff
@@ -167,7 +167,7 @@ def get_unit_coloring(g: Graph) -> Coloring:
 
 
 
-def generate_neighbour_count_with_color(graph: Graph, current_color: int) -> {}:
+def generate_neighbour_count_with_color(coloring: Coloring, current_color: int) -> {}:
     """
     This methode creates a mapping from a vertex to the amount of neighbours with current_color.
     :param graph: graph which is used for the counting of the neighbours
```

Comment:
Kun je deze vervangen met `:param coloring: coloring used for the counting of the neighbours`

Reply:
Tuurlijk :) Voor jou altijd!